src/co_tools/co_fastq.py

```diff
@@ -0,0 +1,156 @@
+import os
+import re
+import subprocess
+import sys
+from glob import glob
+from pathlib import Path
+
+# local imports
+from .get_logger import LOGGER
+
+
+def get_fastq_pair(dir_path: str = "../data"):
```

Comment:
This really needs a docstring to explain what it's for and why you'd use it and how.